Hymns/Browse/BrowseResultsListView.swift

```diff
@@ -0,0 +1,41 @@
+import SwiftUI
+import UIKit
+
+struct BrowseResultsListView: View {
+
+    @Environment(\.presentationMode) var presentationMode
+    let viewModel: BrowseResultsListViewModel
```

Comment:
Make this a private let and add initializer as we normally do? I have already done this and the UIKit and can just push them to your branch if you agree?

Reply:
Sure, sounds good!. I already removed the UiKit, but you can push the private let + initializer